### PR TITLE
Hotfix #18 Create Business Tests Fix

### PR DESCRIPTION
**Pull request by morento101**

### CODE REVIEW

**Review by pazuzu-ua — APPROVED** [review]

File: beauty/api/tests/test_business.py [hunk 1]
```diff
@@ -89,31 +89,34 @@ def setUp(self) -> None:
...
         self.customer = CustomUserFactory.create()
         self.owner = CustomUserFactory.create()
         self.groups.customer.user_set.add(self.customer)
         self.groups.owner.user_set.add(self.owner)
 
+        self.business1 = BusinessFactory.create(owner=self.owner)
+        self.business2 = BusinessFactory.create(owner=self.owner)
+
         self.valid_create_data = {
             "name": faker.word(), "business_type": faker.word(),
```
Comment (Serhii-Voloshyn) [inline beauty/api/tests/test_business.py]: Move "business_type" to next line

File: beauty/api/tests/test_business.py [hunk 2]
```diff
@@ -89,31 +89,34 @@ def setUp(self) -> None:
...
-            "description": faker.text(), "owner": self.owner.id,
+            "description": faker.text(),
         }
         self.invalid_create_data = {
             "name": faker.word(), "business_type": faker.word(),
```
Comment (Serhii-Voloshyn) [inline beauty/api/tests/test_business.py]: Move "business_type" to next line

File: beauty/api/urls.py [hunk 3]
```diff
@@ -4,12 +4,14 @@
...
-from .views_api import (AllServicesListView, BusinessDetailRUDView,
-                        BusinessesListCreateAPIView, CustomUserDetailRUDView,
-                        CustomUserListCreateView, PositionListCreateView, ReviewAddView,
-                        ServiceUpdateView)
 
+from .views_api import (BusinessesListCreateAPIView,
+                        CustomUserDetailRUDView, CustomUserListCreateView,
+                        BusinessDetailRUDView,
+                        ReviewAddView, ServiceUpdateView, AllServicesListView,
+                        PositionListCreateView)
```
Comment (Serhii-Voloshyn) [inline beauty/api/urls.py]: Imports must be in alphabetical order

File: beauty/api/views_api.py [hunk 4]
```diff
@@ -7,29 +7,33 @@
...
 
 from djoser.views import UserViewSet as DjoserUserViewSet
 
-from .models import (Business, CustomUser, Service, Position)
-from .permissions import (IsAccountOwnerOrReadOnly,
-                          IsAdminOrThisBusinessOwner,
-                          IsPositionOwner,
-                          IsProfileOwner,
-                          ReadOnly)
+from .models import Business, CustomUser, Service, Position
```
Comment (Serhii-Voloshyn) [inline beauty/api/views_api.py]: Imports must be in alphabetical order

File: beauty/api/views_api.py [hunk 5]
```diff
@@ -7,29 +7,33 @@
...
+
+from .permissions import (IsAccountOwnerOrReadOnly, IsOwner,
+                          IsPositionOwner, ReadOnly,
+                          IsAdminOrThisBusinessOwner, IsProfileOwner)
```
Comment (Serhii-Voloshyn) [inline beauty/api/views_api.py]: Imports must be in alphabetical order

File: beauty/api/tests/test_business.py (hunk 1 above)
Comment (morento101) [reply beauty/api/tests/test_business.py]: Row length is less than 79 charactes

image

File: beauty/api/urls.py (hunk 3 above)
Comment (morento101) [reply beauty/api/urls.py]: I refuse to order them

File: beauty/api/views_api.py (hunk 4 above)
Comment (morento101) [reply beauty/api/views_api.py]: I refuse to order them

File: beauty/api/views_api.py (hunk 5 above)
Comment (morento101) [reply beauty/api/views_api.py]: I refuse to order them

**Comment by morento101** [comment]
I don't know in which order English letters go in the alphabet, sorry.
I think it is useless to order imports.

I am trying as hard as possible to stick to the flake8 code style, and I do this. Every line has a required length

**Comment by morento101** [comment]
Done ordering and fixed dicts in tests

**Review by Serhii-Voloshyn — APPROVED** [review]

**Review by pazuzu-ua — APPROVED** [review]

**Review by YaroslavBorysko — APPROVED** [review]

**Pull request merged**